docs/components/search-agent.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Search Agent
+description: Documentation for the correct implementation of the Search Agent.
+group: components
+---
+
+Small and adaptive tag for adding context to just about any content.
+
+## Contents
+
+* Will be replaced with the ToC, excluding the "Contents" header
+{:toc}
+
+## Example
+
+Default Search Agent in horizontal alignment
+
+{% example html %}
+<div class="card">
+  <div class="card-block search-agent">
+    <div class="row">
+      <div class="col-12 col-sm-6 push-sm-6">
+        <div class="search-agent-body">
+          <p class="search-agent-subtitle">Suchagent</p>
```

Comment:
I'm not to happy about the naming here, "... subtitle" being mentioned before the "title". Maybe "subtitle" could be "tag" or "byline"